python/lsst/daf/butler/datastores/fileLikeDatastore.py

```diff
@@ -62,6 +62,9 @@
 
 log = logging.getLogger(__name__)
 
+# String to use when a Python None is encountered
+NULLSTR = "__NULL_STRING__"
```

Comment:
@TallJimbo I am pondering whether this should be shorter. Will people want to use this constant elsewhere at some point?

Reply:
Let's not worry about it.  I think if we did need to use sentinal empty strings in more places, we'd move more than just the constant into `Database`, and I don't have any anticipated use cases of that right now.